quil-rs/src/program/snapshots/quil_rs__program__graph__tests__custom_handler__instructionblock_build.snap

```diff
@@ -0,0 +1,126 @@
+---
+source: quil-rs/src/program/graph.rs
+expression: block.graph
+---
+{
```

Comment:
Since these snapshots are pretty inscrutable to a human, I've actually used the dotfile output to test graph construction. It does require the graphviz feature to be set, but when there's a diff it's something that you can copy and paste into a graphviz viewer to get a sense for what materially changed. I'd recommend doing that here as well - there are a couple dozen existing test cases to work from and a useful macro